bin/bridge-client/src/modes/operator/task_manager.rs

```diff
@@ -290,5 +290,8 @@ where
         .await
         .map_err(|e| ExecError::Broadcast(e.to_string()))?;
 
+    info!(%txid, "broadcasted fully signed transaction");
+    debug!(?signed_tx, "broadcasted fully signed transaction");
```

Comment:
Double log here?

Reply:
This is intentional because we want to view the full transaction when debugging. Will demote to `trace`.